project-cli/src/templates/components/class.mustache

```diff
@@ -27,7 +27,7 @@ class {{fileName}} extends React.Component<{{interfaceName}}.IProps, {{interface
 
 	public render(): JSX.Element {
 		return (
-			<div className="{{fileName}}">
+			<div className="{{styleName}}">
```

Comment:
Since page uses this template, when we generate a page, this styleName returns null, therefore page's div don't have any className.